lib/trento.pm

```diff
@@ -518,6 +519,44 @@ sub k8s_logs {
             }
         }
     }
+
+    trento_support();
+}
+
+=head3 trento_support
+
+Call trento-support.sh and dump_scenario_from_k8.sh
+and upload the logs
+=cut
+
+sub trento_support {
+    my $machine_ip = get_trento_ip();
+
+    return unless ($machine_ip);
+
+    my $log_dir = 'remote_logs/';
+    enter_cmd "mkdir $log_dir";
+    my $scp_cmd = sprintf 'scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i %s %s@%s',
+      SSH_KEY, VM_USER, $machine_ip;
+    my $cmd = join(' ',
+        'sudo',
+        'KUBECONFIG=/etc/rancher/k3s/k3s.yaml', 'PATH=${PATH}:/home/' . VM_USER . '/bin/',
+        './trento-support.sh',
+        '--output', 'file-tgz',
+        '--collect', 'all');
+    script_run(az_vm_ssh_cmd($cmd, $machine_ip));
+    script_run("$scp_cmd:'*.tar.gz' $log_dir");
+    my @files = split("\n", script_output('ls -1 ' . $log_dir . '*.tar.gz | cat'));
```

Comment:
Is `| cat` really necessary here? I thought it was not needed due to `ls -1`

Reply:
Inspired by https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/ff698892d36ba624a7e616363b5dc599f219aae6/lib/opensusebasetest.pm#L1283

I'm testing without  cat

Reply:
Weird. I would usually add a `| cat` to an `ls` command if I want to treat the output with a file per line, i.e.:
```
acarvajal@linux-mkji:~/tmp> ls
hola  mundo  robot
acarvajal@linux-mkji:~/tmp> ls | cat
hola
mundo
robot
```
But `ls -1` already does that AFAICT:
```
acarvajal@linux-mkji:~/tmp> ls -1
hola
mundo
robot
```
The man page even says:
```
-1     list one file per line.  Avoid '\n' with -q or -b
```